src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -2,13 +2,14 @@ package racingcar.domain
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
 
 internal class CarTest {
 
     @Test
-    fun `자동차의 전진 조건에 만족하여 전진을 하는 경우`() {
+    fun `자동차의 전진 조건인 랜덤 숫자 4이상이 되어 만족하여 전진을 하는 경우`() {
```

Comment:
함수명을 잘 개선해주셨지만, 너무 자세하게 적어주셔서 오히려 잘 이해가 되지 않는 것 같네요. 🤔 
```suggestion
    fun `숫자가 4 이상이면_전진합니다`() {
```
이와 같이 작성해보면 어떨까요? (이하 관련 내용 동일)